Tests/NetworkServiceTests/HTTPMethodTests.swift

```diff
@@ -0,0 +1,16 @@
+import NetworkService
+import XCTest
+
+final class HTTPMethodTests: XCTestCase {
+    func testAllCasesExistInAPI() {
```

Comment:
I'd suggest to use the [CaseIterable](https://developer.apple.com/documentation/swift/caseiterable) protocol and improve your tests.

Currently, if we add a new HTTP method, the tests won't cover the case.

Reply:
Fixed